elements/rh-pagination/rh-pagination-lightdom.css

```diff
@@ -0,0 +1,131 @@
+rh-pagination {
+  --_link-size: 50px;
+  --_border-color: #b8bbbe;
+  --_border-width: 0;
+  --_border-radius: 0;
+  --_border-accent-width: 0;
+}
+
+rh-pagination > ol {
+  list-style-type: none;
+  display: contents;
+}
+
+rh-pagination li {
+  display: contents;
+}
+
+rh-pagination li > a {
+  text-decoration: none;
+  color: black;
```

Comment:
fixed. if it's not too difficult, please refer to token names instead of values, I had to look these up. not a huge deal, but it helps, and if the values change later, the comment will still make sense.